src/components/BaseClasses/ListBase.js

```diff
@@ -47,6 +47,8 @@ class ListBase extends PureComponent {
 
   render() {
     throw new Error('Not Implemented');
+    // eslint-disable-next-line
```

Comment:
eslint disable comment should be inlined and specific error not just disable.

For Example, `var thing = new Thing(); // eslint-disable-line no-use-before-define`

Reply:
https://github.com/Bnei-Baruch/mdb-admin/pull/62/commits/cae584ea7f75a7e7e03e28163284786c8215b522